content/docs/09-registries-and-packs/4.5-create-pack.md

```diff
@@ -0,0 +1,718 @@
+
+-  **cluster.tf** - has the configuration for `spectrocloud_cluster_aws` resource. The cluster resource depends upon the `spectrocloud_cluster_profile` resource. 
+
+- **data.tf** - contains the configuration for the resources to retrieve data from Palette dynamically. For deploying the `spectrocloud_cluster_profile` resource, here are the pack details for each infrastructure layer. 
+
+  |**Pack Type**|**Registry**|**Pack Name**|**Tag**| **Version** |
+  |---|---|---|---|
+  |OS|Public Repo|`ubuntu-aws`|`LTS__20.4.x`| `20.04`|
+  |Kubernetes|Public Repo|`kubernetes`|`1.24.x`| `1.24.10` |
+  |Network|Public Repo|`cni-calico`|`3.25.x`|`3.25.0`|
+  |Storage|Public Repo|`csi-aws-ebs`|`1.16.x`|`1.16.0`|
+
+  Note that the Terraform code, when executed, will deploy the resources to the **AWS** cloud service provider. To deploy your resource to Azure or Google Cloud, use the specific layer details outlined in the **Cloud Service Provider Configurations** section [here](https://github.com/spectrocloud/tutorials/tree/main/terraform/pack-tf/README.md). 
+
+
+- **inputs.tf** - contains all the Terraform variables and the default values used in the tutorial. Before running the terraform commands, verify and update the default value for all variables. In particular,  update the value for the `cluster_cloud_account_aws_name`, `ssh_key_name`, and `private_pack_registry` variables. You will find the `# ToDo` tags against the fields you want to update. Leave the remaining fields as default, assuming you will continue deploying to the AWS cloud. 
+
+- **outputs.tf** - contains the output variables to expose information.
+
+- **provider.tf** - contains the provider configuration and version.
+
+<br /> <br />
+
+## Deploy Terraform
+After updating the **inputs.tf** file, and carefully reviewing the other files, initialize the Terraform provider:
+<br />
+
+```bash
+terraform init
+```
+
+The `init` command above will download plugins and providers from the **provider.tf** file. Next, preview the resources Terraform will create. 
+<br />
+
+```bash
+terraform plan
+```
+
+The output displays the resources Terraform will create in an actual implementation. Lastly, finish creating all resources using the following command. 
+<br />
+
+```bash
+terraform apply -auto-approve
+```
+
+It will take up to 20 mins to provision a cluster. See a sample successful deplyment in the visual below. 
+While the deployment is in progress, you can switch to the Palette to validate the in-progress deployment. 
+
+<br /> <br />
+
+
+## Check the In-Progress Deployment
+Log back into the [Palette console](https://console.spectrocloud.com/), and navigate to the **Profile** section in the left **Main Menu**. If the terraform deployment is successful, you will see the newly created Profile, as shown in the visual below. 
```

Comment:
Again, guide the reader. There are steps missing. Hold their hand. This should have mentioned to select their cluster so they can track the cluster deployment progress in the cluster detail page.